projects/core/components/alert/alert.component.ts

```diff
@@ -62,11 +56,7 @@ export class TuiAlertComponent<O, I> implements OnInit {
             return;
         }
 
-        timer(
-            tuiIsNumber(this.autoClose)
-                ? this.autoClose
-                : this.options.defaultAutoCloseTime,
-        )
+        timer(tuiIsNumber(this.autoClose) ? this.autoClose : 3000)
```

Comment:
breaking change ?

Reply:
I highly doubt somebody used that. There is already a prop to set autoClose duration.